<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGraphicsView</class>
 <widget class="QWidget" name="MainGraphicsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGraphicsView" name="m_mainGraph">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="NoBrush">
     <color alpha="100">
      <red>50</red>
      <green>51</green>
      <blue>100</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QTreeWidget" name="m_overViewWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>200</width>
     <height>300</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">System OverView</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_scenNext">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>590</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>review</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_scenBack">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>630</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Back To Menu</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="m_fileBrowser">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="m_logo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>341</width>
     <height>341</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/iGaME.images/ustc_software-600*600.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_loadbase">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>195</width>
     <height>59</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>load Database</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_getStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>195</width>
     <height>59</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Get Started</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_runDemo">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>192</width>
     <height>59</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Run Demo</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>biobrick</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>compartment</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="com_combx"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="iGaME.qrc"/>
 </resources>
 <connections/>
</ui>
